FAQ's.md

```diff
@@ -0,0 +1,30 @@
+**Q: What is ADSdroid?**
+
+A: It is an unofficial AllDataSheet Android Application. It gives her users the privilege of searching and downloading datasheets for electronic components in PDF format, for easier and faster access 
```

Comment:
`gives her users the privilege of searching and downloading` should be `makes it possible for users to search and download`